library/src/main/java/com/medium/highlightjs/models/ColorSet.kt

```diff
@@ -0,0 +1,3 @@
+package com.medium.highlightjs.models
+
+data class ColorSet(val mine: String = "#85F8CA", val others: String= "#E5FDF3")
```

Comment:
Do we want hardcoded values here considering we want to support both light and dark modes?

Reply:
those are default values you can see we set them when configuring it it the real app. Maybe I can remove values from here and set them in the sample activity?

Reply:
Yes it would be clearer to force users of this data class to define the colors they want IMO.